Applications/CommandLineInterface/Demo/Scripts_Windows/IdentityTransformRegistration.bat

```diff
@@ -0,0 +1 @@
+..\SuperElastix.exe --loglevel trace --conf ..\Configuration\IdentityTransformRegistration.json --in FixedImage=..\Data\coneA2d64.mhd MovingImage=..\Data\coneB2d64.mhd --out DisplacementField=IdentityDeformationField.mhd
```

Comment:
We should also have an .sh-file.

Reply:
Yes we should. Other thing that is missing is that "make demo" doesn't install this one (and other scripts). Related "request" is that these "demo" scripts are not tested continuously. We could work out the Integration Tests as all "demo" script files being executed.